### PR TITLE
Fix Scheduling Quartz Tasks with MSSQL JDBC Store

### PR DESCRIPTION
**Pull request by Vinche59**
The current implementation does not recognize mssql driver and thus does not load the correct delegate class.

### CODE REVIEW

File: extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ private String guessDriver(Optional<DataSourceDriverBuildItem> dataSourceDriver)
             return HSQLDBDelegate.class.getName();
         }
 
-        if (resolvedDriver.contains("com.microsoft.sqlserver.jdbc.SQLServerResource")) {
+        if (resolvedDriver.contains("com.microsoft.sqlserver.jdbc")) {
```
Comment (machi1990) [inline extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java]: Good catch. Maybe we can make it `com.microsoft.sqlserver.jdbc.SQLServerDriver` instead?

**Review by machi1990 — CHANGES_REQUESTED** [review]

File: extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java (hunk 1 above)
Comment (Vinche59) [reply extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java]: Indeed you're right.
Will do

Comment (machi1990) [reply extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java]: Thanks. Can we squash the two commits into one, please! 
Once that is done, this PR should be good to be merged.

**Review by machi1990 — APPROVED** [review]

**Comment by machi1990** [comment]
merged, thanks @Vinche59

**Pull request merged**